src/test/java/net/sf/jabref/logic/importer/fetcher/DoiFetcherTest.java

```diff
@@ -0,0 +1,82 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.util.Optional;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.BibLatexEntryTypes;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class DoiFetcherTest {
+
+    private DoiFetcher fetcher;
+    private BibEntry bibEntryBurd2011, bibEntryDecker2007;
+
+    @Before
+    public void setUp() {
+        fetcher = new DoiFetcher(Globals.prefs.getImportFormatPreferences());
```

Comment:
Could you pls fix the failing test by using `mock(ImportFormatPreferences.class)` here (if possible, if not `JabRefPreferences.getInstance` or so does the trick)